CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+
+Follow either of the two links above to access the appropriate CLA and instructions for how to sign and return it. Once we receive it, we'll be able to accept your pull requests.
+
+***NOTE***: Only original source code from you and other people that have signed the CLA can be accepted into the main repository.
+
+### Contributing code
+
+If you have improvements to OpenSesame, send us your pull requests! For those
+just getting started, Github has a
+[how to](https://help.github.com/articles/using-pull-requests/).
+
+OpenSesame team members will be assigned to review your pull requests. Once the
+pull requests are approved by two team members and pass continuous integration
+checks, an OpenSesame member will squash and merge your pul request with the 
+master branch. After approval and merging, the site will be redeployed on a 
+quaterly basis. This is done to ensure that we get a good amount of progress 
+onto the website before new versions are rolled out.
+
+If you want to contribute, start working through the OpenSesame codebase,
+navigate to the
+[Github "issues" tab](https://github.com/googleinterns/open-sesame/issues) and start
+looking through interesting issues. If you are not sure of where to start, then
+start by trying one of the smaller/easier issues here i.e.
+[issues with the "good first issue" label](https://github.com/googleinterns/open-sesame/labels/good%20first%20issue)
+and then take a look at the
+[issues with the "contributions welcome" label](https://github.com/googleinterns/open-sesame/labels/stat%3Acontributions%20welcome).
+. If you want to get acquainted with the code first, It might be useful to
+start by looking into our documentation and proposing/fixing some 
+[issues with the "documentation" label](https://github.com/googleinterns/open-sesame/issues?q=is%3Aissue+label%3Adocumentation).
+These are issues that we believe are particularly well suited for outside
+contributions, often because we probably won't get to them right now and it is 
+easier to find holes in documentation if you are new to the codebase. 
+
+***NOTE***: If you decide to start on an issue, leave a comment so that other people know that
+you're working on it. If you want to help out, but not alone, use the issue
+comment thread to coordinate.
+
+### Contribution guidelines and standards
+
+Before sending your pull request for
+[review](https://github.com/googleinterns/open-sesame/pulls),
+make sure your changes are consistent with the guidelines and follow the
+OpenSesame coding style.
+
+#### Coding style
+
+* [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)
+* [Google JavaScript Style Guide](https://google.github.io/styleguide/jsguide.html)
+* [Google HTML/CSS Style Guide](https://google.github.io/styleguide/htmlcssguide.html)
+
+#### Running sanity check and tests
+
+##### Running sanity check
```

Comment:
Is it worth mentioning 'mvn clean' and 'npm install' here because they have helped us solve many otherwise mysterious issues?

Reply:
good point! It skipped my mind. It would make sense to have a common troubleshooting section

Reply:
should this be here or in the `README`?

Reply:
added it to the README